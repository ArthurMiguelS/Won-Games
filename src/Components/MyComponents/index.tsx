import * as S from './styles'

const MyComponents = () => (
  <S.Wrapper>
    <h1>MyComponents</h1>
  </S.Wrapper>
)

export default MyComponents
