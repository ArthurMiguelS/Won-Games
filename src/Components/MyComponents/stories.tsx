import { Meta, StoryObj } from '@storybook/react'
import MyComponents from '.'

export default {
  title: 'MyComponents',
  component: MyComponents
} as Meta

export const Default: StoryObj = {}
