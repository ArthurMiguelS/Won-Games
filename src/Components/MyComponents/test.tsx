import { render, screen } from '@testing-library/react'

import MyComponents from '.'

describe('<MyComponents />', () => {
  it('should render the heading', () => {
    const { container } = render(<MyComponents />)

    expect(screen.getByRole('heading', { name: /MyComponents/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
