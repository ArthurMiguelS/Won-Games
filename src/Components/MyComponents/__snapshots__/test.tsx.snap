// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponents /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    MyComponents
  </h1>
</main>
`;
